Makefile.core.mk

```diff
@@ -145,6 +145,10 @@ export ISTIO_ENVOY_MACOS_RELEASE_DIR ?= ${TARGET_OUT}/release
 export ISTIO_ENVOY_MACOS_RELEASE_NAME ?= envoy-${ISTIO_ENVOY_MACOS_VERSION}
 export ISTIO_ENVOY_MACOS_RELEASE_PATH ?= ${ISTIO_ENVOY_MACOS_RELEASE_DIR}/${ISTIO_ENVOY_MACOS_RELEASE_NAME}
 
+# Variable for WebAssembly plugins
+export STATS_FILTER_WASM_URL ?= $(ISTIO_ENVOY_BASE_URL)/stats-$(ISTIO_ENVOY_VERSION).wasm
```

Comment:
We should just have a “list_of_plugins” var instead of specific variables per plugin.

Reply:
Simplified the variables with a for loop.